src/ports/rentals/components.ts

```diff
@@ -207,8 +212,71 @@ export function createRentalsComponent(
     return parsedResult
   }
 
+  async function getRentalAssets(
+    filters: GetRentalAssetFilters
+  ): Promise<RentalAsset[]> {
+    function getSubgraphQueryOptions() {
+      const queryOptions: string[] = []
+
+      const where: string[] = []
+
+      function mapListToFilter(xs: string[] | undefined, name: string) {
+        if (xs && xs.length > 0) {
+          if (xs.length === 1) {
```

Comment:
As above, I think this case is not neccessary, the `_in` will cover it as well.

Reply:
👍